packages/did-ethr-resolver/src/utils/crypto.ts

```diff
@@ -7,6 +7,8 @@ const {
   keccak256, hashMessage, arrayify, computePublicKey, recoverPublicKey,
 } = utils;
 
+export const compressedSecp256k1KeyLength = 66;
```

Comment:
What do you think if declare this in `constant.ts` instead ? Does that make sense ?

Reply:
Thanks for the suggestion. That could make sense, will keep it in mind.